### DISCUSSION TITLE
Offshore Wind Turbine Simulation

### DISCUSSION THREAD

**Discussion by hamidMech** [Exawind/amr-wind#1253] [Q&A] [answered]
Hello, are there any test cases available that simulate a floating (or fixed-bottom) wind turbine, including wave interactions, using the AMR-Wind simulation environment coupled with OpenFAST?

**Comment by michaelasprague** [comment, marked as answer]
I don't believe we have that case in our examples/tests.  The capability should be there, however, to do an AMR-Wind/OpenFAST simulation of a floating turbine using an actuator-line rotor representation and where the rotor motion due to floating motion is calculated via hydrodyn.

We are currently developing features to do high-fidelity simulation of floating turbines with the full ExaWind solver (AMR-Wind in the background; Nalu-Wind resolving near turbine/platform fluid).  That is work in progress.
